docs/networking.md

```diff
@@ -10,8 +10,10 @@ help you out.
 
 {% include /style/icon.html type="info" %}
 For Raspberry Pi platforms with no display, you must use the wired Ethernet port
-to connect for the first time. You can set up additional connections using the
-`connmanctl` command through a remote SSH session.
+to connect for the first time through a remote SSH session. If you connected an HDMI 
```

Comment:
Can we change this sentence to clarify that this is another option, e.g. "You can alternately connect an..."?